[2025-01-07T02:55:22.299+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-07T02:55:22.313+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ico_scraping_dag_new_records.store_in_postgres manual__2025-01-07T02:52:58.795128+00:00 [queued]>
[2025-01-07T02:55:22.324+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ico_scraping_dag_new_records.store_in_postgres manual__2025-01-07T02:52:58.795128+00:00 [queued]>
[2025-01-07T02:55:22.324+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-07T02:55:22.341+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): store_in_postgres> on 2025-01-07 02:52:58.795128+00:00
[2025-01-07T02:55:22.351+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=6176) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-07T02:55:22.352+0000] {standard_task_runner.py:72} INFO - Started process 6179 to run task
[2025-01-07T02:55:22.351+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ico_scraping_dag_new_records', 'store_in_postgres', 'manual__2025-01-07T02:52:58.795128+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/ico_scraping_dag_insert_new_records.py', '--cfg-path', '/tmp/tmpmco4edp2']
[2025-01-07T02:55:22.354+0000] {standard_task_runner.py:105} INFO - Job 32: Subtask store_in_postgres
[2025-01-07T02:55:22.401+0000] {task_command.py:467} INFO - Running <TaskInstance: ico_scraping_dag_new_records.store_in_postgres manual__2025-01-07T02:52:58.795128+00:00 [running]> on host e0f05d443e24
[2025-01-07T02:55:22.481+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ico_scraping_dag_new_records' AIRFLOW_CTX_TASK_ID='store_in_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-01-07T02:52:58.795128+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-07T02:52:58.795128+00:00'
[2025-01-07T02:55:22.483+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-07T02:55:22.523+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2025-01-07T02:55:22.528+0000] {sql.py:553} INFO - Running statement: SELECT project_link FROM "ICODATA";, parameters: None
[2025-01-07T02:55:22.530+0000] {sql.py:562} INFO - Rows affected: 100
[2025-01-07T02:55:22.535+0000] {logging_mixin.py:190} INFO - No new records to insert
[2025-01-07T02:55:22.536+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-07T02:55:22.544+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-07T02:55:22.544+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ico_scraping_dag_new_records, task_id=store_in_postgres, run_id=manual__2025-01-07T02:52:58.795128+00:00, execution_date=20250107T025258, start_date=20250107T025522, end_date=20250107T025522
[2025-01-07T02:55:22.568+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-07T02:55:22.581+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-07T02:55:22.583+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
